content/applications/finance/accounting/vendor_bills/deferred_expenses.rst

```diff
@@ -1,195 +1,196 @@
+decide to recognize it on a monthly basis. Each month, for the next 12 months, $100 will be
 recognized as an expense.
 
-Odoo Accounting handles deferred expenses and prepayments by spreading them in multiple entries that
-are automatically created in *draft mode* and then posted periodically.
+Odoo Accounting handles deferred expenses by spreading them across multiple entries that are
+posted periodically.
 
 .. note::
    The server checks once a day if an entry must be posted. It might then take up to 24 hours before
    you see a change from *draft* to *posted*.
 
-Prerequisites
+Configuration
 =============
 
-Such transactions must be posted on a **Deferred Expense Account** rather than on the default
-expense account.
+Make sure the default settings are
+correctly configured for your business. To do so, go to :menuselection:`Accounting --> Configuration
+--> Settings`. The following options are available:
+
+Journal
+  The deferral entries will be posted in this journal.
+Deferred Expense Account
+  Expenses will be deferred on this Current Asset account until they are recognized.
+Deferred Revenue Account
+  Revenues will be deferred on this Current Liability account until they are recognized.
+Generate Entries
+  By default, Odoo will :ref:`automatically generate <vendor_bills/deferred/generate_on_validation>`
+  the deferral entries when you post a vendor bill. However,
+  you can also choose to :ref:`generate them manually <vendor_bills/deferred/generate_manually>`
+  by selecting the :guilabel:`Manually & Grouped` option instead.
+Amount Computation
+  Suppose an bill of $1200 must be deferred over 12 months. The :guilabel:`Equal per month`
+  computation will recognize for $100 each month. While the :guilabel:`Based on days`
+  computation will recognize for different amounts depending on the number of days in each month.
+
+.. _vendor_bills/deferred/generate_on_validation:
+
+Generate deferral entries on validation
+=======================================
 
-Configure a Deferred Expense Account
-------------------------------------
+.. tip::
+  Make sure the :guilabel:`Start Date` and :guilabel:`End Date` fields are visible in the
+  :guilabel:`Invoice Lines` tab.
 
-To configure your account in the **Chart of Accounts**, go to :menuselection:`Accounting -->
-Configuration --> Chart of Accounts`, click on *Create*, and fill out the form.
+For each line of the bill, dated in January, that should be deferred, specify the
+start and end dates of the deferral period.
 
-.. image:: deferred_expenses/deferred_expenses01.png
-   :align: center
-   :alt: Configuration of a Deferred Expense Account in Odoo Accounting
+.. example::
+  You can defer a purchase of $1200 over 12 months by specifying a start date of 01/01/2023 and an
+  end date of 12/31/2023.
 
-.. note::
-   This account's type must be either *Current Assets* or *Prepayments*
+If the :guilabel:`Generate Entries` field is set to `On invoice/bill validation`, Odoo will
```

Comment:
Sorry I didn't catch this on my first read review - for `On invoice/bill validation` we want to use a :guilabel: instead of a `literal` because it's a selection field in Odoo. If it were a text field where you had to manually enter a value, that's when you would use the `literal` format.

There are 2 instances of this on the expenses and revenue docs: one here and one around line 103 with "Manually & Grouped".